ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/qos-profiles.j2

```diff
@@ -12,20 +12,22 @@ QOS Profile: **{{ profile }}**
 
 | Default COS | Default DSCP | Trust |
 | ----------- | ------------ | ----- |
-{%         set cos = qos_profiles[profile].cos | default('-') %}
-{%         set dscp = qos_profiles[profile].dscp | default('-') %}
-{%         set trust = qos_profiles[profile].trust | default('-') %}
+{%         set cos = qos_profiles[profile].cos | arista.avd.default('-') %}
+{%         set dscp = qos_profiles[profile].dscp | arista.avd.default('-') %}
+{%         set trust = qos_profiles[profile].trust | arista.avd.default('-') %}
 | {{ cos }} | {{ dscp }} | {{ trust }} |
 
 **Tx-queues**
 
-{%         if qos_profiles[profile].tx_queues is defined and qos_profiles[profile].tx_queues is not none %}
+{%         if qos_profiles[profile].tx_queues is arista.avd.defined %}
 | Tx-queue | Bandwidth | Priority |
 | -------- | --------- | -------- |
 {%             for tx_queue in qos_profiles[profile].tx_queues | arista.avd.natural_sort %}
-{%                 set queue = qos_profiles[profile].tx_queues[tx_queue] | default('-') %}
-{%                 set bw_percent = qos_profiles[profile].tx_queues[tx_queue].bandwidth_percent | default('-') %}
-{%                 set priority = qos_profiles[profile].tx_queues[tx_queue].priority | default('-') %}
+{%                  set queue = qos_profiles[profile].tx_queues[tx_queue] | arista.avd.default('-') %}
+{%                  set bw_percent = qos_profiles[profile].tx_queues[tx_queue].bandwidth_perfect | arista.avd.default(
```

Comment:
typo, `perfect` instead of `percent`. This causes the unintended changes in the molecule output.